doc/charm++/manual.rst

```diff
@@ -2879,6 +2879,36 @@ load balancing strategies, including ScotchLB that relies on the
 external partitioning library SCOTCH specified in the
 Section :numref:`lbOption`.
 
+In order to retrain the Random Forest model in Metabalancer, the training
+dataset needs to be generated. Following 28 features are to be used:
+
+- *pe_imbalance*: max_pe_load/avg_pe_load
```

Comment:
There are some inconsistencies in capitalization, separators, etc. in this table that should be fixed.